tests/Fixer/ReturnNotation/ReturnAssignmentFixerTest.php

```diff
@@ -620,6 +620,178 @@ function Foo() {
                     }
                 ',
             ],
+            'try catch' => [
+                '<?php
+                function foo()
+                {
+                    if (isSomeContiotion()) {
```

Comment:
I know that this "typo" is from my suggestion, but you might want to fix it as it will be your name in `git blame` for it 😛